eclair-core/src/test/scala/fr/acinq/eclair/channel/states/a/WaitForAcceptChannelStateSpec.scala

```diff
@@ -114,29 +114,29 @@ class WaitForAcceptChannelStateSpec extends TestkitBaseClass with StateTestsHelp
   test("recv AcceptChannel (reserve below dust limit)") { f =>
     import f._
     val accept = bob2alice.expectMsgType[AcceptChannel]
-    val reserveTooSmall = accept.dustLimitSatoshis - 1
+    val reserveTooSmall = accept.dustLimitSatoshis.toLong - 1
```

Comment:
Isn't `-` defined for `Satoshi` and `Millisatoshi`?
It seems to me that we could (should) get rid of most calls to `toLong` and leverage supported operators and implicit conversions...